src/components/annotations/index.js

```diff
@@ -343,8 +343,8 @@ annotations.draw = function(gd, index, opt, value) {
         .classed('annotation-text-g', true)
         .attr('data-index', String(index));
 
-    var ann = anng.append('svg')
-        .call(Plotly.Drawing.setPosition, 0, 0);
+    var ann = anng.append('g')
+        .attr('transform', 'translate(0,0)');
```

Comment:
Oh. I meant, `'transform', 'translate(0,0)'` as no effects, right?

By the why, I like the idea of having a `drawing.setTranslate` method. No need to do that right now though.